### PR TITLE
docs: replace contentBase by static

### PR DESCRIPTION
**Pull request by i-oliva**
`contentBase` has been deprecated by `static`. This changes the docs accordingly so people will be able to read up-to-date docs.

### CODE REVIEW

**Review by chenxsan — DISMISSED** [review]

File: examples/module-federation/app1/webpack.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   entry: "./src/index",
   mode: "development",
   devServer: {
-    contentBase: path.join(__dirname, "dist"),
+    static: path.join(__dirname, "dist"),
```
Comment (snitin315) [inline examples/module-federation/app1/webpack.config.js:10]: Need to update the webpack-dev-server version to v4 as well in package.json here.

Comment (snitin315) [reply examples/module-federation/app1/webpack.config.js:10]: https://github.com/webpack/webpack.js.org/blob/98973c7bb1fb6ef80b6428e01a9bf597412b075c/examples/module-federation/app1/package.json#L22

https://github.com/webpack/webpack.js.org/blob/98973c7bb1fb6ef80b6428e01a9bf597412b075c/examples/module-federation/app2/package.json#L21

Comment (chenxsan) [reply examples/module-federation/app1/webpack.config.js:10]: Once updated, you can check this link https://stackblitz.com/github/webpack/webpack.js.org/tree/8e4e3207e24932c3f64b44ce0ac8a5e8092605ad/examples/module-federation to see if the example works.

Comment (i-oliva) [reply examples/module-federation/app1/webpack.config.js:10]: lgtm 
image

**Review by chenxsan — APPROVED** [review]

File: examples/module-federation/app1/webpack.config.js (hunk 1 above)
Comment (chenxsan) [reply examples/module-federation/app1/webpack.config.js:10]: Awesome.

**Pull request merged**